ext/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Range for Chrome",
-  "version": "1.0.4",
+  "version": "1.0.5",
```

Comment:
If we're sticking with semver (I think we should), this should get a minor version bump (1.1.0) since this is adding functionality that didn't exist before.

Reply:
Good call.